src/data/program_configuration.js

```diff
@@ -123,40 +117,60 @@ class SourceExpressionBinder<T> implements Binder<T> {
     type: string;
     statistics: { max: number };
 
-    constructor(expression: SourceExpression, name: string, type: string) {
+    PaintVertexArray: Class<StructArray>;
```

Comment:
Not used outside the constructor; make it a local variable and remove the `_omit`.